sdk/monitor/azure-monitor-opentelemetry/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed issue #42337, added support for instrumentations-any to ensure better dependency detection.
```

Comment:
Title is not good enough to understand what is being fixed here, instrumentation_any_deps is very internal details on how OpenTelemetry Instrumentation package works, let's describe what is being fixed instead not how